### PR TITLE
Update to MantisBT 2.6.0 (Resolving the inline script problem)

### PR DESCRIPTION
**Pull request by rmkpatchaa**
I've update the plugin to resolve the inline script problem.
The core modification is that all javascript code is in a separate js file. The configuration is injected via meta tags.
I've also : 

- Update google api php client to latest version
- Added french translation
- Made some cleanup

### CODE REVIEW

**Comment by luzpaz** [comment]
@kkremitzki perhaps we can test this on FC ?

**Comment by dam326** [comment]
don't work with 2.8.0 . 
have you planned an update soon ?

**Comment by rmkpatchaa** [comment]
Hi @dam326 I didn't test it with 2.8 version since it was not yet published when I  pushed this update. Can you debug and tell which error you have ?

**Comment by luzpaz** [comment]
@rmkpatchaa take a look at https://github.com/mantisbt-plugins/GoogleOauth/issues/3#issuecomment-342388232

**Comment by rmkpatchaa** [comment]
@luzpaz ok. Unfortunately I don't have time for now to do it. I'll check later if I can. It should be good if someone here has a little time to dig into it.

**Comment by rmkpatchaa** [comment]
@luzpaz I think there is a misunderstanding... Dependencies for plugins given as examples are MantisBT 2.3+. 
My fix works for MantisBT 2.6. 
So if there is a problem with 2.8 release, something might have changed since then.

**Comment by xezpeleta** [comment]
Hi! I've applied the patch and it works like a charm with Mantis 2.8. Can anybody merge this PR ?

@luzpaz have you configured the plugin with the correct credentials?

Thanks @rmkpatchaa !! Good work!!

**Comment by luzpaz** [comment]
I haven't tested yet. I need to ask @kkremitzki if he can install this plugin, though I know he's busy these days.  
I wonder if @dregad is interested in this ?

**Comment by dregad** [comment]
> I wonder if @dregad is interested in this ?

Why should I be ?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created by .ignore support plugin (hsz.mobi)
+### Example user template template
+### Example user template
+
+# IntelliJ project files
+.idea
+*.iml
+out
+gen### Example user template template
+### Example user template
```
Comment (dregad) [inline .gitignore]: @rmkpatchaa Looks like something went wrong with your .gitignore changes, the same files are excluded twice and the comments are duplicated as well.

**Comment by luzpaz** [comment]
@dregad Well I thought it would be a nice feature to be able to tout for Mantis, especially that most of the popular bugtrackers provide this feature not just for GitHub but other bugtrackers as well.

**Comment by dregad** [comment]
So you mean integrating the plugin as a standard authentication method in the core ? If yes, this is what @vboctor's comment https://github.com/mantisbt-plugins/GoogleOauth/issues/3#issuecomment-342388232 is about.

**Comment by dam326** [comment]
@xezpeleta :  i test it again and it works ! i don't understand why it didn't work my first time.

**Comment by luzpaz** [comment]
What's blocking this from merge ?

File: .gitignore (hunk 1 above)
Comment (rmkpatchaa) [reply .gitignore]: Edited :)

**Comment by comxd** [comment]
In my case (MantisBT 2.10) the htaccess located in /plugins folder prevent access to the plugin.js file located at /plugins/GoogleOauth/pages/assets/js/plugin.js

**Pull request merged**